### PR TITLE
Fixing Invisibles Drawing Issues for Right-to-Left Writing Direction

### PR DESCRIPTION
**Pull request by HassanTaleb90**
Screenshot 2024-08-12 at 5 30 03 AM

### CODE REVIEW

File: Sources/STTextViewAppKit/STTextLayoutFragment.swift [hunk 1]
```diff
@@ -104,31 +102,33 @@ final class STTextLayoutFragment: NSTextLayoutFragment {
...
+                    case 0x000A: "\u{00AC}"  // ¬ Line Feed
+                    case 0x000D: "\u{21A9}"  // ↩ Carriage Return
+                    case 0x00A0: "\u{235F}"  // ⎵ Non-Breaking Space
+                    case 0x200B: "\u{205F}"  // ⸱ Zero Width Space
+                    case 0x200C: "\u{200C}"  // ‌ Zero Width Non-Joiner
+                    case 0x200D: "\u{200D}"  // ‍ Zero Width Joiner
+                    case 0x2060: "\u{205F}"  //   Word Joiner
+                    case 0x2028: "\u{23CE}"  // ⏎ Line Separator
+                    case 0x2029: "\u{00B6}"  // ¶ Paragraph Separator
+                    default: "\u{00B7}"  // • Default symbol for unspecified whitespace
```
Comment (krzyzanowskim) [inline Sources/STTextViewAppKit/STTextLayoutFragment.swift:117]: what’s with that formatting changes?

Comment (HassanTaleb90) [reply Sources/STTextViewAppKit/STTextLayoutFragment.swift:117]: Tabs to spaces

Comment (HassanTaleb90) [reply Sources/STTextViewAppKit/STTextLayoutFragment.swift:117]: Screenshot 2024-08-12 at 10 31 09 AM

**Review by krzyzanowskim — APPROVED** [review]
thank you

**Pull request merged**